pkg/tools/cli/loadtest/usecase/load.go

```diff
@@ -18,27 +18,52 @@ package usecase
 
 import (
 	"context"
+	"fmt"
+	"strings"
 
+	"github.com/vdaas/vald/internal/client/gateway/vald/grpc"
 	"github.com/vdaas/vald/internal/errgroup"
 	"github.com/vdaas/vald/internal/runner"
-	"github.com/vdaas/vald/pkg/agent/ngt/config"
+	"github.com/vdaas/vald/pkg/tools/cli/loadtest/config"
+	"github.com/vdaas/vald/pkg/tools/cli/loadtest/service"
+	"github.com/vdaas/vald/pkg/tools/cli/loadtest/service/insert"
+	"github.com/vdaas/vald/pkg/tools/cli/loadtest/service/search"
 )
 
 type run struct {
 	eg  errgroup.Group
 	cfg *config.Data
+	l   service.Load
 }
 
 func New(cfg *config.Data) (r runner.Runner, err error) {
-	return nil, nil
+	run := new(run)
+	run.cfg = cfg
+	run.eg = errgroup.Get()
+
+	ctx := context.Background()
+	c, err := grpc.New(ctx, grpc.WithAddr(cfg.Address)) // TODO setup vald grpc client
+	if err != nil {
+		return nil, fmt.Errorf("grpc connection error")
+	}
+	switch strings.ToLower(cfg.Method) {
+	case "insert":
```

Comment:
It may be better `const` variable :thinking:

e.g)
```go
switch operation.Atoo(strings.ToLower(cfg.Method)) {
 	case operation.Insert:
 	case operation.Search:
 	default
}

-- type package or operation package??:thinking:

type Operation uint8

const (
	Insert Operation = iota
	Search 
)

func Atoo(str string) Operation {
....
}

func String() string {
...
}

```